test/introspection/ERC165Checker.test.js

```diff
@@ -33,9 +33,10 @@ contract('ERC165Checker', function (accounts) {
       expect(supported).to.equal(false);
     });
 
-    it('does not support mock interface via supportsAllInterfaces', async function () {
-      const supported = await this.mock.supportsAllInterfaces(this.target.address, [DUMMY_ID]);
-      expect(supported).to.equal(false);
+    it('does not support mock interface via getSupportedInterfaces', async function () {
```

Comment:
Please don't remove the existing test and add the new one in a separate `it` just like you did for the other contexts